services/baseremotecontrol/builtin/builtin_test.go

```diff
@@ -32,6 +32,13 @@ func TestBaseRemoteControl(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	testutils.VerifySameElements(t, depNames, []string{"baseTest", "inputTest"})
 
+	cfg.ControlModeName = "fail"
+	_, err = cfg.Validate("")
+	test.That(t, err, test.ShouldBeError,
+		errors.New(
+			//nolint: lll
+			`Error validating. Path: "" Error: Control mode 'fail' is not in [joystickControl triggerSpeedControl buttonControl arrowControl droneControl]`))
```

Comment:
I wasn't sure about the best way to break up this line to appease the linter that didn't break the test, i.e. using a multi-line string. I'm open to suggestions.